app/upgrades.go

```diff
@@ -30,3 +44,67 @@ func (app *App) registerUpgrade0_1_4(_ upgradetypes.Plan) {
 		},
 	)
 }
+
+func (app *App) registerUpgrade0_2_0(upgradeInfo upgradetypes.Plan) {
+	const planName = "v0.2.0"
```

Comment:
It looks like it might not be required for us: 
https://github.com/cosmos/ibc-go/blob/v6.0.0/docs/migrations/v5-to-v6.md (5 -> 6 requires one if your chain has a custom authentication module using the ICS27 controller submodule)
https://github.com/cosmos/ibc-go/blob/v7.0.0/docs/migrations/v6-to-v7.md (6 -> 7 has an optional upgrade handler to prune expired tendermint consensus states)

Reply:
👍 👍